deploy/postgresql.yaml

```diff
@@ -56,7 +56,7 @@ spec:
   ports:
     - port: 5432
       targetPort: 5432
-      nodePort: 31032
+      # nodePort: 31032
```

Comment:
Should we rather separate this service into prod, and dev services for Postgres? Instead of commenting out this line.

Reply:
Let it be this way for now and we can do that in the future depending on the requirements!